packages/manager/src/features/linodes/LinodesDetail/LinodeRebuild/RebuildFromImage.tsx

```diff
@@ -203,6 +248,32 @@ export const RebuildFromImage: React.FC<CombinedProps> = (props) => {
                 disabled={disabled}
                 passwordHelperText={passwordHelperText}
               />
+              {shouldDisplayUserDataAccordion ? (
+                <>
+                  <Divider spacingTop={40} />
+                  <UserDataAccordion
+                    userData={userData}
+                    onChange={handleUserDataChange}
+                    disabled={reuseUserData}
+                    renderNotice={() => (
+                      <StyledNotice
+                        success
+                        text="Adding new user data is recommended as part of the rebuild process."
+                      />
+                    )}
+                    renderCheckbox={() => (
+                      <Box>
+                        <CheckBox
+                          checked={reuseUserData}
```

Comment:
Can we use a more descriptive variable name for `reuseUserData`? Maybe something like `shouldReuseUserData`? reuseUserData sounds more like a function name to me.

Also what makes us need to use state here over the formik form?

```suggestion
                          checked={reuseUserData}
```

Reply:
With regards to the state for `checked`, since we wouldn't want to submit `shouldReuseUserData` as part of the payload, I think we're in a similar position here as we were in for `AccessKeyDrawer.tsx`